[WIP] vr&e

dannychapman opened the pull request:
WIP, do not merge.

dannychapman committed:
Re-stubbed VR&E pages

dannychapman committed:
Merge branch 'master' into vre-content-revised

dannychapman committed:
Merge branch 'master' into vre-content-revised

dannychapman committed:
Merge branch 'master' into vre-content-revised

dannychapman committed:
Merge branch 'master' into vre-content-revised

dannychapman committed:
Merge branch 'master' into vre-content-revised

goldenmeanie committed:
Merge branch 'master' into vre-content-revised

goldenmeanie committed:
Merge branch 'master' into vre-content-revised

goldenmeanie committed:
Merge branch 'master' into vre-content-revised

dannychapman committed:
Merge branch 'master' into vre-content-revised

dannychapman committed:
Merge branch 'master' into vre-content-revised

dannychapman committed:
Merge branch 'master' into vre-content-revised

dannychapman committed:
Merge branch 'master' into vre-content-revised

dannychapman committed:
Updated folder, link title for VRE section

rroueche committed:
Merge branch 'master' into vre-content-revised

jcmeloni-usds commented:
@dannychapman Is this still a WIP from Sept/Oct or should it be abandoned? cc @ayaleloehr

dannychapman commented:
@rroueche What's the status on this one? Safe to remove at this point?

rroueche commented:
Checking w/Beth bc I don't remember the reasoning for why this got put on hold

@bethpotts do you know if we have a plan for this content, or if it just got put on hold indefinitely? Maybe we need to check back in with SMEs and/or Mary Ann?

Commit:
Merge branch 'master' into vre-content-revised

# Conflicts:
#	content/pages/education/advanced-training-and-certifications.md
#
content/pages/education/advanced-training-and-certifications/entrepreneu
rship-training.md
#
content/pages/education/advanced-training-and-certifications/flight-trai
ning.md
#
content/pages/education/advanced-training-and-certifications/licensing-c
ertification.md
#
content/pages/education/advanced-training-and-certifications/national-te
sting-program.md
#	content/pages/education/apply-for-education-benefits.md
#	content/pages/education/gi-bill.md
#	content/pages/education/gi-bill/buy-up-program.md
#	content/pages/education/gi-bill/foreign-programs.md
#	content/pages/education/gi-bill/higher-learning.md
#	content/pages/education/gi-bill/montgomery-active-duty.md
#	content/pages/education/gi-bill/montgomery-selected-reserve.md
#	content/pages/education/gi-bill/post-9-11.md
#	content/pages/education/gi-bill/survivors-dependent-assistance.md
#
content/pages/education/gi-bill/survivors-dependent-assistance/dependent
s-education.md
#
content/pages/education/gi-bill/survivors-dependent-assistance/fry-schol
arship.md
#	content/pages/education/gi-bill/transfer.md
#	content/pages/education/gi-bill/tuition-assistance.md
#	content/pages/education/gi-bill/tutorial-assistance.md
#	content/pages/education/gi-bill/yellow-ribbon.md
#	content/pages/education/index.md
#	content/pages/education/other-educational-assistance-programs.md
#
content/pages/education/other-educational-assistance-programs/call-to-se
rvice.md
#	content/pages/education/other-educational-assistance-programs/reap.md
#	content/pages/education/other-educational-assistance-programs/veap.md
#	content/pages/education/rates.md
#	content/pages/education/tools-programs.md
#	content/pages/education/tools-programs/education-career-counseling.md
#	content/pages/education/tools-programs/locate-a-school.md
#
content/pages/education/tools-programs/principles-excellence-program.md
#	content/pages/education/work-learn.md
#	content/pages/education/work-learn/co-op-training.md
#	content/pages/education/work-learn/job-and-apprenticeship.md
#	content/pages/education/work-learn/non-college-degree-program.md
#	content/pages/education/work-learn/non-traditional.md
#
content/pages/education/work-learn/non-traditional/accelerated-payments.
md
#
content/pages/education/work-learn/non-traditional/correspondence-traini
ng.md
#
content/pages/education/work-learn/non-traditional/independent-distance-
learning.md
#	content/pages/education/work-learn/workstudy.md

Commit:
Merge branch 'master' into vre-content-revised

Commit:
fixing typo

Commit:
updating claim types URL

Commit:
updating incorrect file path

DanielleSoCompany committed:
PL edit

Beth: A few other PL edits I would suggest if we want to do a slightly deeper pass:
- change "limits your ability to work" to "makes it harder for you to work"
- change "receive additional training" to "get more job or skills training"
- break this up into 2 sentences so that "in some cases your family members" comes out and becomes a second sentence (to avoid an overly long sentence): In some cases, your family members can get these benefits too.

DanielleSoCompany committed:
PL edit

Beth: Let me know what you think of the edits I've made in this one (using it as a test case; for the others, for now I'll just do the contractions and really small things). I think adding definitions where appropriate, making piano keys action oriented where possible without changing meaning, and speaking about VA in the first person (ie, "our" VR&E" programs) are fairly critical to maintaining style throughout site. But I can change back if you think it's too much.

DanielleSoCompany committed:
First pass PL edit

DanielleSoCompany committed:
First pass PL

Beth: I think for most of these, it's okay to have the link within the sentence because it starts the sentence off and is a program name. The exception is the last one, where "a monthly subsistence allowance" was a link right in the middle of the sentence. So I revised that to be "See the current subsistence allowance rates" as a 2nd sentence. Also changed "You can earn" to "You may also be able to earn" because the VA.gov page this goes to says, "In some cases"...so I think "can" would be over-promising there.

DanielleSoCompany committed:
First pass PL

Beth: There's definitely more PL'ing we could do on this page, in particular, but this is my first pass "light touch" per discussion. A few things for your attention:
1) I broke out the Veteran bullet in the eligibility section because it was introducing an "an" bullet that only applied to that specific "or" bullet, which I think is confusing.
2) Ok to change "Who can apply?" to "Who's covered?" to maintain consistency across pages, or does this one need to remain "Who can apply?" 
3) Lots of "eligible"s here if we end up deciding we can change that.
4) In step 6, we tell them to fill out the form, but we don't tell them what to do with it. Should we add something about where to take/mail it?

DanielleSoCompany committed:
First pass PL

Beth: Definitely more PL'ing we could do here, but here's my first pass "light touch". Note that when I added a comma after the coding for %, it seemed to mess up the coding. Looking into how to fix that.

DanielleSoCompany committed:
Deeper PL Pass

Notes for Beth:
1) In "Servicemembers" PK: Moved the "if" conditional phrase to the beginning of the sentence for low-literacy users' easier ability to understand whether it applies to them.
2) Following up on our conversation re: how best to handle "service-connected disability": I looked that up on the VA.gov site and this is how it was worded there: "Disability compensation is a monetary benefit paid to Veterans who are determined by VA to be disabled by an injury or illness that was incurred or aggravated during active military service. These disabilities are considered to be service connected." So I've added a PL version of this as a separate sentence that reads: "Having a service-connected disability means we've concluded that you were disabled by an injury or illness that started or got worse during active military service." If you agree this works, I can update the glossary and we can update throughout the site as a next step. Thanks!

DanielleSoCompany committed:
Deeper PL Pass

DanielleSoCompany committed:
Deeper PL Pass

DanielleSoCompany committed:
Deeper PL Pass 2

DanielleSoCompany committed:
Deeper PL Pass

Beth: I noticed the "How do I get these benefits?" section was a bit varied across the subpages; some go more into process, while others don't. Making some changes for consistency, and leaving the step-by-step approach to the actual Apply page. Let me know if you disagree with this approach.

DanielleSoCompany committed:
Deeper PL Pass

DanielleSoCompany committed:
Deeper PL Pass - Ready for Beth's Review

Beth: This is the page where we have questions for SMEs re: application process.

DanielleSoCompany committed:
PL Pass - Ready for Beth Review

DanielleSoCompany committed:
Deeper PL Pass - Ready for Beth Review

DanielleSoCompany committed:
Deeper PL Pass - Ready for Beth Review

Beth - 2 notes:
1) I'm not sure if the bulleted list is working within the subway map. May need to ask Mel about that after I can see it in heroku.
2) Changed "home-modification programs" to "adaptive-housing programs" because the Apply pages uses the "adaptive-housing" language and that's also what we use in the Housing Assistance section. Please correct me if that's wrong here. Thanks!.

DanielleSoCompany committed:
Deeper PL Pass - Ready for Beth Review

DanielleSoCompany committed:
Deeper PL Pass - Ready for Beth Review

DanielleSoCompany committed:
Deeper PL Pass - Ready for Beth Review

Note for Beth: The "How do I get these benefits?" section was just 1 sentence about the business plan, which didn't feel right. So I added some language about applying based on other pages and then just added a conclusion sentence.

bethpotts committed:
Initial review: apply-vre

DanielleSoCompany committed:
Fixing code

Commit:
Change template, fix layout and markdown formatting issues

melwoodard commented on content/pages/vocational-rehab-and-employment/apply-vre.md:
```diff
@@ -1,26 +1,11 @@
+template: 6-info-page
 ---
 
-<div class="main" role="main" markdown="0">
```
@bethpotts you shouldn't need any of this gunk at the top of a template, just start off right with intro text. This is what was making the column so narrow.

melwoodard commented on content/pages/vocational-rehab-and-employment/apply-vre.md:
```diff
@@ -1,26 +1,11 @@
-
-
-
-<div markdown="1">
```
Removing unnecessary divs surrounding content. This can get pretty in the weeds, but a general guideline is you should only need to put div tags before and after a chunk of content when you're doing some type of special styling like a blue callout box or subway map.

melwoodard commented on content/pages/vocational-rehab-and-employment/apply-vre.md:
```diff
@@ -48,59 +33,63 @@ You may also qualify for VR&amp;E benefits if you're a Veteran who was discharge
+<ol class="process">
+
+<li class="step one">
+
```
The reason the markdown wasn't formatting correctly here - it needs an extra empty line above and below the text.

melwoodard commented on content/pages/vocational-rehab-and-employment/apply-vre.md:
```diff
@@ -48,59 +33,63 @@ You may also qualify for VR&amp;E benefits if you're a Veteran who was discharge
 </li>
 
-<li class="step two wow fadeIn animated" markdown="1">
+<li class="step two">
```
For the class names for subway map steps, this is all you should need: `<li class="step {number}">` and if it's the last step, `<li class="step last {number}">`

bethpotts committed:
Proof: vre landing page

bethpotts committed:
Update family-members.md

bethpotts committed:
First review fixing code: independent-living

rroueche committed:
updating file names (#5301)

Commit:
Update overview page intro text and button formatting

Commit:
Format fix: /external-resources

Commit:
Format fix: /service-disabled

Commit:
Format fix: /vetsuccess

Commit:
Format fix: /service-disabled/existing-skills

Commit:
Merge remote-tracking branch 'origin/vre-content-revised' into vre-content-revised

Commit:
Format fix: new-skills, return-to-job, small-business-training

Commit:
Fix subway map formatting on /independend-living

bethpotts committed:
Proof 2: VRE landing page

Removed definition of service disabled

bethpotts committed:
Proof: service-disabled

bethpotts committed:
Proof: apply-vre

bethpotts committed:
Changed ebenefits link: family-members

bethpotts committed:
Proof: job-search-and-counseling

bethpotts committed:
Proof 2: job-search-and-counseling

bethpotts committed:
Moved job seekers link down: job-search-and-counseling

bethpotts committed:
Tried to add space before blue box: job-search-and-counseling

bethpotts committed:
Proof: new-skills

bethpotts committed:
Proof: small-business-training

bethpotts committed:
Proof 2 fix blue box: new-skills

bethpotts committed:
Fix 2 blue box: new-skills

bethpotts committed:
Fixed blue box: job-search-and-counseling

bethpotts committed:
Proof: independent-living

bethpotts committed:
Add blue box: independent-living

bethpotts committed:
Add Who's covered: independent-living

bethpotts committed:
Proof: return-to-job

DanielleSoCompany committed:
Updates based on IA discussion

DanielleSoCompany committed:
Updates per IA discussion - Ready for Beth Review

DanielleSoCompany committed:
Updates per IA discussion

Beth: I added in an explanation of service-connected disability and disability rating now that this page speaks to eligibility-specific info.  And then I bolded the lead-ins so that they stood out from the notes more. Thoughts?

DanielleSoCompany committed:
Updates per IA discussion

DanielleSoCompany committed:
Updates per IA discussion

DanielleSoCompany committed:
Updates per IA discussion

DanielleSoCompany committed:
Updates per IA discussion

DanielleSoCompany committed:
Updates per IA discussion

DanielleSoCompany committed:
Updates per IA discussion

DanielleSoCompany committed:
Fixing relatedlinks code

bethpotts committed:
Proof: vetsuccess

Commit:
Fix relatedlinks indentation

Commit:
Merge remote-tracking branch 'origin/vre-content-revised' into vre-content-revised

bethpotts committed:
Proof: external-resources

Commit:
fix broken link

Commit:
Add space above feature box

bethpotts committed:
Removed comma: vetsuccess

DanielleSoCompany committed:
Adding related links

DanielleSoCompany committed:
Adding related links

DanielleSoCompany committed:
adding related links

DanielleSoCompany committed:
adding related links

DanielleSoCompany committed:
adding related links

DanielleSoCompany committed:
adding related links

DanielleSoCompany committed:
adding related links

DanielleSoCompany committed:
adding related links

DanielleSoCompany committed:
fixing card title

DanielleSoCompany committed:
small fixes

DanielleSoCompany committed:
small fixes

DanielleSoCompany committed:
small fixes

DanielleSoCompany committed:
small fixes

DanielleSoCompany committed:
small fixes

DanielleSoCompany committed:
small fixes

DanielleSoCompany committed:
small fixes

DanielleSoCompany committed:
fixing extra indents in related links

DanielleSoCompany committed:
adding related links

DanielleSoCompany committed:
adding related links

DanielleSoCompany committed:
fixing spacing with blue box

DanielleSoCompany committed:
fixing card 1 title

DanielleSoCompany committed:
updating piano keys

DanielleSoCompany committed:
trying to fix spacing in subway map

DanielleSoCompany committed:
fixing first card for spacing

DanielleSoCompany committed:
fixing 1st card for spacing

DanielleSoCompany committed:
fixing piano keys

DanielleSoCompany committed:
fixing piano keys

DanielleSoCompany committed:
fixing piano keys

DanielleSoCompany committed:
fixing piano keys

Beth: I also took out the note re: family members at the end and added a "Qualified family members" link to the "Who's covered?" section to align with other pages.

DanielleSoCompany committed:
fixing intro & piano keys

Beth: There were a few funky things in the intro that I also fixed so please do a full proof of this page.

DanielleSoCompany committed:
fixing spelling error

DanielleSoCompany committed:
fixing piano keys

DanielleSoCompany committed:
fixing spelling error

DanielleSoCompany committed:
fixing wording & piano key

bethpotts committed:
Proof: landing page

bethpotts committed:
Proof: apply-vre

bethpotts committed:
Removed space before the apply link: new-skills

bethpotts committed:
Removed space before apply link: small-business-training

DanielleSoCompany committed:
application process updates

DanielleSoCompany committed:
updating disability rating definition

DanielleSoCompany committed:
updating definition of disability rating

DanielleSoCompany committed:
updating disability rating definition

DanielleSoCompany committed:
updating subway map

bethpotts committed:
Fix typo: family-members

bethpotts committed:
Bolding: family-members

bethpotts committed:
Update index.md

bethpotts committed:
Added blue card for vr&e

bethpotts committed:
Remove bolding: apply-vre

Commit:
Merge branch 'master' into vre-content-revised

Commit:
Merge of master including Heroku fixes

Commit:
Fixing merge to master

bethpotts committed:
Title change to blue card for VR&E on Empl landing page

DanielleSoCompany committed:
Incorporating SME changes

DanielleSoCompany committed:
Incorporating SME changes

DanielleSoCompany committed:
Incorporating SME changes

DanielleSoCompany committed:
Incorporating SME changes

DanielleSoCompany committed:
Incorporating SME changes/tweaks

DanielleSoCompany committed:
Content tweaks

DanielleSoCompany committed:
Incorporating SME changes

DanielleSoCompany committed:
Incorporating SME changes

DanielleSoCompany committed:
Incorporating SME changes

DanielleSoCompany committed:
Incorporating SME changes

DanielleSoCompany committed:
Incorporating SME changes

DanielleSoCompany committed:
Updating track names and descriptions

DanielleSoCompany committed:
Updating track names and descriptions

DanielleSoCompany committed:
Updating track names and descriptions

DanielleSoCompany committed:
updating a track description

DanielleSoCompany committed:
updating page title

DanielleSoCompany committed:
updating track names and descriptions

DanielleSoCompany committed:
updating track names and descriptions

DanielleSoCompany committed:
updating track names and descriptions

DanielleSoCompany committed:
updating track names and descriptions

DanielleSoCompany committed:
minor edit

DanielleSoCompany committed:
removed predischarge claim related link

DanielleSoCompany committed:
Finalizing SME changes

DanielleSoCompany committed:
Finalizing SME changes

DanielleSoCompany committed:
Finalizing SME changes take 2

DanielleSoCompany committed:
Finalizing SME changes

DanielleSoCompany committed:
adding note

DanielleSoCompany committed:
Finalizing SME changes

DanielleSoCompany committed:
changing "assistance" to "support"

DanielleSoCompany committed:
Finalizing SME changes

DanielleSoCompany committed:
Finalizing SME changes

DanielleSoCompany committed:
Finalizing SME changes

DanielleSoCompany committed:
Finalizing SME changes

DanielleSoCompany committed:
Finalizing SME changes

DanielleSoCompany committed:
updating title

DanielleSoCompany committed:
related link description tweak

DanielleSoCompany committed:
related link description tweak

DanielleSoCompany committed:
related link description tweak

DanielleSoCompany committed:
Incorporating SME changes

bethpotts committed:
Proof: VRE landing page

bethpotts committed:
Proof/ME review: service-disabled

bethpotts committed:
Proof and ME review: job-search-and-counseling

bethpotts committed:
Proof and ME review: new-skills

bethpotts committed:
Proof and ME review: small-business-training

bethpotts committed:
Proof and ME review: independent-living

bethpotts committed:
Proof and ME review: independent-living

bethpotts committed:
Proof and ME review: return-to-job

bethpotts committed:
Proof and ME review: family-members

bethpotts committed:
Proof and ME review: vetsuccess

DanielleSoCompany committed:
Revising intro per conversation with Beth

DanielleSoCompany committed:
incorporating proofing tweak

DanielleSoCompany committed:
Revising per conversation with Beth

DanielleSoCompany committed:
Revising intro per conversation with Beth

DanielleSoCompany committed:
Revising intro per conversation with Beth

DanielleSoCompany committed:
Revising per conversation with Beth

DanielleSoCompany committed:
Adding IDES as part of VR&E update

DanielleSoCompany committed:
minor update

DanielleSoCompany committed:
adding blue box

DanielleSoCompany committed:
minor update

DanielleSoCompany committed:
adding link to predischarge and IDES

DanielleSoCompany committed:
adding space

DanielleSoCompany committed:
Removing internal resources & updating links

DanielleSoCompany committed:
trying to create space between button and box

DanielleSoCompany committed:
Updates per discussion with Amy

DanielleSoCompany committed:
additional tweaks

DanielleSoCompany committed:
updating title of programs page

DanielleSoCompany committed:
updates

ayaleloehr commented:
If i recall, VR&E went in, so should this be closed?

rroueche commented:
@bethpotts @leannammiller @amybehrens this was the old branch for VR&E work, it's probably obsolete now, so yes - per Alex's comment I think it can be deleted, but wanted to check with you all first.

patrickvinograd commented:
Way obsolete. Closing.
